variables.tf

```diff
@@ -52,4 +48,4 @@ variable "consul_server_cluster_size" {
 variable "consul_client_cluster_size" {
   description = "The number of nodes to have in the Consul Client example cluster. Any number of nodes is permissible, though 3 is usually enough to test.."
   default = 3
-}
```

Comment:
I believe `var.update_strategy` was removed in `main.tf`. Is it still needed in `vars.tf`?